framework_tvb/tvb/core/entities/model/model_datatype.py

```diff
@@ -113,6 +113,40 @@ class DataType(HasTraitsIndex):
     fk_from_operation = Column(Integer, ForeignKey('OPERATIONS.id', ondelete="CASCADE"))
     parent_operation = relationship(Operation, backref=backref("DATA_TYPES", order_by=id, cascade="all,delete"))
 
+    @property
+    def summary_info(self):
+        # type: () -> typing.Dict[str, str]
+
+        ret = {}
+        if self.title:
+            ret['Title'] = str(self.title)
+
+        columns = self.get_base_table_columns
+        for attribute in columns:
+            try:
+                attr_field = getattr(self, attribute)
+                attr_name = self.get_attribute_name(attribute)
+                if attr_name is not None:
+                    ret[attr_name] = str(attr_field)
+            except Exception:
+                pass
+        return ret
+
+    @property
+    def get_base_table_columns(self):
+        columns = self.__table__.columns.keys()
+        if type(self).__bases__[0] is DataType:
+            return columns
+        base_table_columns = type(self).__bases__[0].__table__.columns.keys()
+        columns.extend(base_table_columns)
+        return columns
+
+    def get_attribute_name(self, attr_name):
```

Comment:
In general a new method is not a bad idea, but in this case, I suggest to inline the code and  don't put this part in another method, because the `return None` convention is odd and a bit of an antipattern